app/src/main/java/org/schabi/newpipe/util/PicassoHelper.java

```diff
@@ -139,21 +140,23 @@ public Bitmap transform(final Bitmap source) {
                                         .getDimension(R.dimen.player_notification_thumbnail_width),
                                 source.getWidth());
 
-                        final Bitmap result = Bitmap.createScaledBitmap(
+                        final Bitmap result = BitmapCompat.createScaledBitmap(
                                 source,
                                 (int) notificationThumbnailWidth,
                                 (int) (source.getHeight()
                                         / (source.getWidth() / notificationThumbnailWidth)),
+                                null,
                                 true);
 
-                        if (result == source) {
+                        if (result == source && !result.isMutable()) {
```

Comment:
What I meant with

> create a new thumbnail both if result == source and if !result.isMutable()

was `||`, not `&&`, sorry for the confusion.
```suggestion
                        if (result == source || !result.isMutable()) {
```